msbuild/Xamarin.iOS.Tasks.Windows/Xamarin.iOS.Tasks.Windows.csproj

```diff
@@ -4,19 +4,23 @@
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <MergeSystemAssemblies>false</MergeSystemAssemblies>
+    <RuntimeIdentifier>win</RuntimeIdentifier>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="DotNetZip" Version="1.14.0" />
+    <PackageReference Include="Mono.Cecil" Version="0.11.3" />
```

Comment:
latest is 0.11.4
https://www.nuget.org/packages/Mono.Cecil/0.11.4

Reply:
I'll update it

Reply:
Done, check latest two commits